### PR TITLE
Add breadcrumbs and header elements, refactor routes, link restaurants to restaurant reviews page

### PR DESCRIPTION
**Pull request by oxyflush**
This MR:
1. Refactors existing app.js routes. This is done primarily to place them in a more common, cleaner file that is able to reuse and export constants that will be needed for other elements.
2. Adds breadcrumbs from reactstrap and header elements to the restaurants page. These elements pull from the routes.js file that specifies aliases for some pages and provides emphasis on active breadcrumbs with links on the inactive ones.
3. Provide a link to restaurant review page for each restaurant.

Addresses https://github.com/NickPriv/FryRank/issues/18

### CODE REVIEW

File: frontend/src/components/Common/Breadcrumb.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { React } from 'react'
+import { useLocation } from 'react-router-dom'
+import { Breadcrumb, BreadcrumbItem } from 'reactstrap'
+import { pathToPageName } from '../../routes'
+
+const Breadcrumbs = () => {
```
Comment (NickPriv) [inline frontend/src/components/Common/Breadcrumb.js:6]: Nit: Rename the file or the method name for consistency

File: frontend/src/components/Restaurants/index.jsx [hunk 2]
```diff
@@ -11,12 +15,15 @@ const propTypes = {
...
 
     const restaurantsDisplay = (restaurants) => {
-        return restaurants.map((restaurant, i) => (
-            <Fragment key = {i}>
-                <p><b>{restaurant.name}</b> </p>
-                <AddressDisplay address={restaurant.address} />
-            </Fragment>
-        ));
+        return restaurants.map((restaurant, i) => {
+            let restaurantLink = `http://${window.location.hostname}:3000${PATH_REVIEWS}`.replace(PATH_VARIABLE_RESTAURANT_ID, restaurant.id)
```
Comment (NickPriv) [inline frontend/src/components/Restaurants/index.jsx:19]: Let's make a constant for the hostname / port #

File: frontend/src/containers/Restaurants/index.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ const mapDispatchToProps = {
     getRestaurants: restaurantsActions.startGetRestaurantsRequest
 };
 
-export default compose(
+export const RestaurantsContainer = compose(
```
Comment (NickPriv) [inline frontend/src/containers/Restaurants/index.js:17]: Try using "Restaurants" as the export name so you don't have to create another constant to export

**Review by NickPriv — CHANGES_REQUESTED** [review]

File: frontend/src/containers/Restaurants/index.js (hunk 3 above)
Comment (oxyflush) [reply frontend/src/containers/Restaurants/index.js]: Done in [270a580](https://github.com/NickPriv/FryRank/pull/21/commits/270a5806712ecc2080dc3256cf5936d155029a8e).

File: frontend/src/components/Restaurants/index.jsx (hunk 2 above)
Comment (oxyflush) [reply frontend/src/components/Restaurants/index.jsx]: Done in [270a580](https://github.com/NickPriv/FryRank/pull/21/commits/270a5806712ecc2080dc3256cf5936d155029a8e).

File: frontend/src/components/Common/Breadcrumb.js (hunk 1 above)
Comment (oxyflush) [reply frontend/src/components/Common/Breadcrumb.js]: Done in [1863c90](https://github.com/NickPriv/FryRank/pull/21/commits/1863c901244d5475537db2b6e7b7465160bc4aad).

**Review by NickPriv — APPROVED** [review]

**Pull request merged**